nav2_system_tests/src/system/test_system_launch.py

```diff
@@ -122,6 +148,26 @@ def generate_launch_description():
         ]
     )
 
+def find_gz_sim_processes():
```

Comment:
I recommend actually moving these into a utils directory in the simple commander package. I think these will be useful in a number of tests and applications

Reply:
Make sense

Reply:
done